SECP-service/src/test/java/com/visucius/secp/models/MessageTest.java

```diff
@@ -2,11 +2,17 @@
 
 import static org.junit.Assert.*;
 
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
 import org.hibernate.validator.constraints.Email;
 import org.junit.Assert;
 import org.junit.Test;
 
 import javax.persistence.*;
+import java.sql.Timestamp;
```

Comment:
is there a reason why you included these import statements ? I don't think you are using them in your test. :)

Reply:
Oops, nice catch. I was trying some stuff and forgot to remove them.